pkg/kubectl/cmd/util/cmdtools.go

```diff
@@ -73,34 +81,40 @@ func defaultEnvShell() []string {
 	return []string{shell, flag}
 }
 
-func defaultEnvEditor(envs []string) ([]string, bool) {
-	var editor string
+func defaultEnv(cmdName string, envs []string) ([]string, bool) {
```

Comment:
> cmdName string

I suggest you define a type for this to avoid random string get passed in.
```
type CmdName string
const(
  editorCmd CmdName = "editor"
  diffCmd CmdName = "diff"
)
```